doc/user/formats.rst

```diff
@@ -130,3 +130,66 @@ fractional_ranges
         be masked if :math:`[\frac{i-0.5}{nchans}, \frac{i+0.5}{nchans}]`
         overlaps any of the ranges. Note that this means that channel 0
         is centred at 0.0 but extends below it.
+
+Primary beam
+------------
+The file contains the information in a compact Fourier-transform ("aperture
+plane") form. It comprises a 3-dimensional array J of Jones matrices. Another
+three arrays :math:`\nu`, :math:`y` and :math:`x` indicate the frequency (in
+Hz) and spatial position (in metres) of each sample along the axes.
+
+To determine the response at a sampled frequency :math:`\nu_k` and some
+direction, turn the direction into direction cosines :math:`l` and :math:`m`
+relative to the pointing centre. Then the Jones
+matrix for the response is
+
+.. math:: \frac{1}{|x|\cdot|y|}
```

Comment:
not sure what you are trying to imply with |x||y| but the scaling factor should be the total number of samples being summed over, not the size of an area.

Reply:
x and y are vectors, so |x| is the length of the vector.

Reply:
I think that is obscure, atleast add a note to say that.

Reply:
Done.